examples/mnist/train_mnist.py

```diff
@@ -8,8 +8,8 @@
 import argparse
 
 import numpy as np
+import pickle
```

Comment:
`pickle` is a standard library. So please move `import` to the top of import group.